tests/data/format/linewrap_summary/max_line_length_50.args

```diff
@@ -0,0 +1,2 @@
+--linewrap-full-docstring
```

Comment:
@mdczaplicki I have pushed some changes. Turns out I was being very dumb.

Since line wrapping is still experimental it is not enabled by default. So for the test to run, it also needs this flag set.

Reply:
No problem. Thanks for the help here :D

Reply:
No worries and thanks very much for doing the initial work. It's much easier as maintainer to build on a PR that's 90% done than starting from scratch.

Sorry for the spam you're getting right now. There is something wrong with one of our test runs but I can only test it on a PR opened by somebody that is not a contributor... I'll clean the commits in this PR up after I have fixed the issue and then merge this!

Reply:
Tests pass locally. :tada: 

BTW. I believe there is a problem with `make html`. I was running it, but pipelines complained.

Also:
```
❯ cd docs
❯ pip install -e .
Obtaining file:///home/marek/PycharmProjects/pydocstringformatter/docs
ERROR: file:///home/marek/PycharmProjects/pydocstringformatter/docs does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
❯ pip install -r requirements-doc.txt
Obtaining file:///home/marek/PycharmProjects/pydocstringformatter/docs (from -r requirements-doc.txt (line 1))
ERROR: file:///home/marek/PycharmProjects/pydocstringformatter/docs (from -r requirements-doc.txt (line 1)) does not appear to be a Python project: neither 'setup.py' nor 'pyproject.toml' found.
```

However it works when I go 
```
cd ..
``` 
and then
```
pip install -r docs/requirements-doc.txt
```

Reply:
Yeah it should be:
```python
pip install -e .
cd docs
make html
```

This is due to how `readthedocs` install dependencies from root. So we need to install them from root as well...